specification/portal/resource-manager/Microsoft.Portal/preview/2019-01-01-preview/portal.json

```diff
@@ -413,19 +413,19 @@
           "description": "The dashboard's part position.",
           "properties": {
             "x": {
-              "type": "number",
+              "type": "integer",
```

Comment:
What's the difference between number and integer

Reply:
1 will become 1.0 when msrest doing serialization which will be reject by service team